pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1023,15 +1040,15 @@ PrequalifierModel:
   type: object
   required:
     - guid
-    - name
+    - title
```

Comment:
What is the requirement behind this change? This is a breaking change to the API and will require client modifications.

Reply:
We can postpone API change. When is a better time?

Reply:
Also, this `PrequalifierModel` is used for this "prequalifer" API which I don't think any studies uses. I'm inclined to remove this API completely.

Reply:
Let me take a look at both the apps and verify that. If neither one is using it, I'm game for adding a ticket to remove the API. One less vestigial endpoint will be nice.

Reply:
Took note of this in DDP-4515. Resolving comment.